pkg/kotsadm/main.go

```diff
@@ -535,6 +558,7 @@ func ensureKotsadm(deployOptions types.DeployOptions, clientset *kubernetes.Clie
 				if err := ensureMinio(deployOptions, clientset); err != nil {
 					return errors.Wrap(err, "failed to ensure minio")
 				}
+				// TODO: This returns before kotsadm-minio is updated
```

Comment:
This appears to return before the minio statefulset gets updated, so it doesn't wait for the initcontainers

Reply:
Updated the login in `ensureMinio` to validate the number of initcontainers before returning

Reply:
no longer relevant.  was able to fix `WaitForStatefulSetReady` by changing to:
```
if s.Status.ObservedGeneration == s.ObjectMeta.Generation &&
	s.Status.ReadyReplicas == *s.Spec.Replicas &&
	s.Status.UpdateRevision == s.Status.CurrentRevision {
	return nil
}
```